lib/aux_file.ml

```diff
@@ -92,7 +93,8 @@ let load_aux_file_for vfile =
   | End_of_file -> !h
   | Sys_error s | Scanf.Scan_failure s
   | Failure s | Invalid_argument s ->
-    Flags.if_verbose Feedback.msg_info Pp.(str"Loading file "++str aux_fname++str": "++str s);
-     empty_aux_file
+    if CDebug.(get_flag misc) then
+      Feedback.msg_info Pp.(str"Loading file "++str aux_fname++str": "++str s);
```

Comment:
```suggestion
    CDebug.pp_misc Pp.(fun () -> str"Loading file "++str aux_fname++str": "++str s);
```

Reply:
For readability it would be nice if OCaml didn't force you to provide the `fun () ->` for every call to the routine.  Maybe something like "FUN" that works like a macro.  Seems like this is a pattern that's used endlessly.